uiowa/uiowa
# [ITS] Service category breadcrumbs

Resolves #6872 





# How to test



`ddev blt frontend`

```
ddev blt ds --site=its.uiowa.edu && ddev drush @its.local config:import --source ../config/sites/its.uiowa.edu --partial -y && ddev drush @its.local cr && ddev drush @its.local uli
```

Add a Page w/title "Services" (specifically, a URL of `/services`). This'll be the overall landing page for Services.

Add the service with no Service Category, check out the breadcrumbs. It should be `Home / Services / <title>`
Add a Service Category taxonomy term, update a Service with that Service Category, and check out the breadcrumbs. It should be ``Home / Services / <service category> / <title>`, with appropriate links.
Check that the Service Category term page URL is of the form `services/<term-name>`

Check other node types and pages (taxonomy terms, views, etc) and that their breadcrumbs are unaffected
